sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1660,6 +1660,14 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val CSV_PARSER_MAX_ERROR_CONTENT_LENGTH = buildConf(
```

Comment:
Can we not make a config for this? it really isn't that important. Just cap at anything reasonable like 1000

Reply:
CC @HyukjinKwon 
I am OK with or without configuration. What do you think ?

Reply:
Update code. Currently I hardcoded the maxErrorContentLength to be 1024. Adding another config looks too fussy for end users. This is just a minor setting.

Reply:
Yea, let's do that then. We can add an option later if users complain. I wonder if higher then 1000 exception message is actually helpful though.